app/src/features/surveys/SurveysLayout.tsx

```diff
@@ -0,0 +1,35 @@
+import Box from '@material-ui/core/Box';
```

Comment:
This file appears to be identical to the ProjectLayout.tsx.  I think for now we can delete this, and just use the ProjectLayout.  

(Even the projectLayout isn't really doing much, but we can address that issue separately later, as needed)